.travis.yml

```diff
@@ -10,7 +10,24 @@ before_install:
   - sudo apt-get install -y python-apt autoconf pkg-config e2fslibs-dev libblkid-dev zlib1g-dev liblzo2-dev asciidoc
 install:
   - sudo pip install ansible==2.3.1
-  - ./scripts/travis.sh install
+  # dash? wtf is dash? UGH! use a real shell
+  - sudo rm /bin/sh
```

Comment:
NACK the intent was not to remove the scripts/travis.sh script. That should remain. It should simply call the make targets for each of its steps.